frontend/src/modules/lf/config/status.js

```diff
@@ -1,21 +1,21 @@
 export default [
   {
-    color: 'bg-green-500',
+    color: 'green',
     label: 'Active',
     value: 'active',
   },
   {
-    color: 'bg-primary-500',
+    color: 'primary',
     label: 'Formation',
     value: 'formation',
   },
   {
-    color: 'bg-yellow-500',
+    color: 'yellow',
     label: 'Prospect',
     value: 'prospect',
   },
   {
-    color: 'bg-gray-400',
+    color: 'gray',
     label: 'Archived',
     value: 'archived',
   },
```

Comment:
Not sure if it's related, but we used to have dots with colors before the options on the dropdown, perhaps this change caused that.
I think this `color`'s purpose is to use directly in the template, and not to use it as a class
![Screenshot 2024-12-04 at 08 10 38](https://github.com/user-attachments/assets/86270770-eee4-4efa-b535-b5d52fbbbc5d)

Reply:
done